Client/Frontend/Browser/SearchLoader.swift

```diff
@@ -53,6 +52,43 @@ class SearchLoader: Loader<Cursor<Site>, SearchViewController>, FeatureFlaggable
         }
     }
 
+    fileprivate func getHistoryAsSites(matchingSearchQuery query: String, limit: Int) -> Deferred<Maybe<Cursor<Site>>> {
```

Comment:
nit; we prefer to use `private` in new code rather than `fileprivate`. Unless specifically needed.
```suggestion
    private func getHistoryAsSites(matchingSearchQuery query: String, limit: Int) -> Deferred<Maybe<Cursor<Site>>> {
```